swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md

```diff
@@ -25,7 +25,7 @@ The code generation tool can produce `ballerina stub` and `ballerina service/cli
 You can generate Ballerina source code using the following command:
 
 ```
-./ballerina grpc --input <proto-file-path> [--output <path>] [--mode client | service]
+$ bal grpc --input <proto-file-path> [--output <path>] [--mode client | service]
```

Comment:
```suggestion
$ bal grpc --input <proto-file-path> [--output <path>] [--mode client | service | proxy]
```
The output of the `bal help grpc` command is as following

![image](https://user-images.githubusercontent.com/9464997/105465114-8c267300-5cb8-11eb-9397-6e7cfb4a9d8c.png)

Reply:
Fixed